src/main/java/hu/bme/mit/spaceship/TorpedoStore.java

```diff
@@ -19,21 +19,19 @@ public boolean fire(int numberOfTorpedos){
       throw new IllegalArgumentException("numberOfTorpedos");
     }
 
-    boolean success = false;
 
     //simulate random overheating of the launcher bay which prevents firing
     double r = generator.nextDouble();
 
     if (r > 0.1) {
       // successful firing
       this.torpedos -= numberOfTorpedos;
-      success = true;
+      return true;
     } else {
       // failure
-      success = false;
+      return false;
```

Comment:
Szia, akár ki is lehetne törölni az else ágat. És akkor mehetne alá defaultból egyreturn false.